src/components/ComingSoon.js

```diff
@@ -0,0 +1,16 @@
+
+const ComingSoon = () => {
+    return (
+      <div className="container">
+        <div className="py-5 text-center">
+          <img src="img/footer/soon.png" height="700px" alt="Stay tuned" />
```

Comment:
Move this to coming-soon folder in images directory.

Reply:
done

Reply:
👍